crates/ingress-kafka/src/subscription_controller.rs

```diff
@@ -259,6 +258,7 @@ mod task_orchestrator {
             let subscription_id = if let Some(subscription_id) =
                 self.running_tasks_to_subscriptions.remove(&task_id)
             {
+                warn!("Consumer unexpectedly closed");
```

Comment:
Do you think we have enough context in current span (I've not checked) to help us/users to figure out what happened?

Reply:
probably not, will add more